pkg/storaged/stratis-panel.jsx

```diff
@@ -51,18 +51,27 @@ function stratis_pool_row(client, path) {
     const pool = client.stratis_pools[path];
 
     return {
-        client, name: pool.Name, key: path, devname: "/dev/stratis/" + pool.Name + "/",
+        client,
+        name: pool.Name,
+        key: path,
+        devname: "/dev/stratis/" + pool.Name + "/",
         detail: cockpit.format(_("$0 Stratis pool"), fmt_size(pool.TotalPhysicalSize)),
-        go: () => cockpit.location.go(["pool", pool.Uuid]), job_path: path
+        go: () => cockpit.location.go(["pool", pool.Uuid]),
+        job_path: path
     };
 }
 
 function stratis_locked_pool_row(client, uuid) {
     const action = <StorageButton onClick={() => unlock_pool(client, uuid, true)}><UnlockIcon /></StorageButton>;
 
     return {
-        client, actions: action, name: uuid, key: uuid, truncate_name: false,
-        detail: _("Locked encrypted Stratis pool"), go: () => cockpit.location.go(["pool", uuid])
+        client,
+        actions: action,
+        name: uuid,
+        key: uuid,
+        truncate_name: false,
+        detail: _("Locked encrypted Stratis pool"),
+        go: () => cockpit.location.go(["pool", uuid])
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221107-162851-fa6168ae-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)